<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>item_editor_dialog</class>
 <widget class="QDialog" name="item_editor_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Item</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3">
    <widget class="localized_spin_box" name="sbSellNPC">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string> gp</string>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Statistics</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QLabel" name="lbBuyMin">
        <property name="text">
         <string>Min:</string>
        </property>
        <property name="buddy">
         <cstring>sbBuyMin</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="localized_spin_box" name="sbBuyMin">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> gp</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="lbSellMin">
        <property name="text">
         <string>Min:</string>
        </property>
        <property name="buddy">
         <cstring>sbSellMin</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="localized_spin_box" name="sbTotalBought">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> gp</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="localized_spin_box" name="sbBuyMax">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> gp</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lbSellMax">
        <property name="text">
         <string>Max:</string>
        </property>
        <property name="buddy">
         <cstring>sbSellMax</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbBuyAvg">
        <property name="text">
         <string>Avg:</string>
        </property>
        <property name="buddy">
         <cstring>sbBuyAvg</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="localized_spin_box" name="sbSold">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> units</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbBought">
        <property name="text">
         <string>Bought:</string>
        </property>
        <property name="buddy">
         <cstring>sbBought</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lbSold">
        <property name="text">
         <string>Sold:</string>
        </property>
        <property name="buddy">
         <cstring>sbSold</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbBuyMax">
        <property name="text">
         <string>Max:</string>
        </property>
        <property name="buddy">
         <cstring>sbBuyMax</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="localized_spin_box" name="sbBought">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> units</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="localized_spin_box" name="sbSellMin">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> gp</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbTotalBought">
        <property name="text">
         <string>Total:</string>
        </property>
        <property name="buddy">
         <cstring>sbTotalBought</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="localized_spin_box" name="sbSellMax">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> gp</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="localized_spin_box" name="sbTotalSold">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> gp</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbLastBought">
        <property name="text">
         <string>Last:</string>
        </property>
        <property name="buddy">
         <cstring>dtLastBought</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="localized_spin_box" name="sbLastBought">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> units</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="lbLastSold">
        <property name="text">
         <string>Last:</string>
        </property>
        <property name="buddy">
         <cstring>dtLastSold</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="localized_spin_box" name="sbLastSold">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> units</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="lbSellAvg">
        <property name="text">
         <string>Avg:</string>
        </property>
        <property name="buddy">
         <cstring>sbSellAvg</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="localized_spin_box" name="sbSellAvg">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> gp</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="lbTotalSold">
        <property name="text">
         <string>Total:</string>
        </property>
        <property name="buddy">
         <cstring>sbTotalSold</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="localized_spin_box" name="sbBuyAvg">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> gp</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="localized_spin_box" name="sbLastTotalBought">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> gp</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="localized_spin_box" name="sbLastTotalSold">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> gp</string>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QDateTimeEdit" name="dtLastBought">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::YearSection</enum>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2" colspan="2">
       <widget class="QDateTimeEdit" name="dtLastSold">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::YearSection</enum>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd HH:mm:ss</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbProfit">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>Do not worth</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/img/do_not_worth.png</normaloff>:/img/do_not_worth.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bad</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/img/bad.png</normaloff>:/img/bad.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/img/medium.png</normaloff>:/img/medium.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Good</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/img/good.png</normaloff>:/img/good.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Excelent</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/img/excelent.png</normaloff>:/img/excelent.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbBuyNPC">
     <property name="text">
      <string>NPC:</string>
     </property>
     <property name="buddy">
      <cstring>sbBuyNPC</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbName">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="buddy">
      <cstring>edName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="edName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Profit:</string>
     </property>
     <property name="buddy">
      <cstring>cbProfit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="lbSellNPC">
     <property name="text">
      <string>NPC:</string>
     </property>
     <property name="buddy">
      <cstring>sbSellNPC</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="localized_spin_box" name="sbBuyNPC">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string> gp</string>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>localized_spin_box</class>
   <extends>QSpinBox</extends>
   <header>localized_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edName</tabstop>
  <tabstop>cbProfit</tabstop>
  <tabstop>sbBuyNPC</tabstop>
  <tabstop>sbSellNPC</tabstop>
  <tabstop>sbBought</tabstop>
  <tabstop>sbBuyMax</tabstop>
  <tabstop>sbBuyAvg</tabstop>
  <tabstop>sbBuyMin</tabstop>
  <tabstop>sbTotalBought</tabstop>
  <tabstop>sbLastBought</tabstop>
  <tabstop>sbLastTotalBought</tabstop>
  <tabstop>dtLastBought</tabstop>
  <tabstop>sbSold</tabstop>
  <tabstop>sbSellMax</tabstop>
  <tabstop>sbSellAvg</tabstop>
  <tabstop>sbSellMin</tabstop>
  <tabstop>sbTotalSold</tabstop>
  <tabstop>sbLastSold</tabstop>
  <tabstop>sbLastTotalSold</tabstop>
  <tabstop>dtLastSold</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>item_editor_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>item_editor_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
